app/lib/utilities.ts

```diff
@@ -0,0 +1,19 @@
+const formatAmountToHumanReadable = (amountStr: string) => {
```

Comment:
[note]

There's `app/lib/utils.js` already, so I named this one `app/lib/utilities.ts`.

I'm thinking of converting the code in `app/lib/utils.js` to TS and putting everything to `app/lib/utils.ts` (then will remove `app/lib/utilities.ts` ).
But let me know if that's unnecessary.